general/general-03.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, Union
```

Comment:
Good job checking the input params. Good implementation, for the for loop you could have used list.sort and slice list[-2]